### ISSUE TITLE
从meta-llama/Llama-2-13b-hf到SparseLLM/ReluLLaMA-13B

### ISSUE THREAD

**Issue by Vincent131499** [SJTU-IPADS/PowerInfer#32]
Good job!
从[meta-llama/Llama-2-13b-hf](meta-llama/Llama-2-13b-hf)到[SparseLLM/ReluLLaMA-13B](https://huggingface.co/SparseLLM/ReluLLaMA-13B)应该要经过一个finetune，看readme里面没有写，这个具体finetune训练会开放吗

**Comment by YixinSong-e** [comment]
Thank you very much for your interest in our project. Regarding the specific fine-tuning of relu-llama, the related code and work will still require some time to be ready. If you are very interested, a basic way to reproduce our fine-tuning process is to simply change the activation function from silu to relu and then start training the model on the corpus. You will observe that the loss rapidly decreases around 1 billion tokens, and as the number of training tokens increases, the model's capabilities will be restored more and more. An important thing to note is that our corpus needs to include as many. :)

非常感谢你对我们的关注，关于relu-llama的具体finetune，相关的代码和工作仍需要等待一段时间。如果你非常感兴趣，一个基本能复现我们finetune的方式就是直接将激活函数从silu改为relu后，开始模型在语料上的训练。你会看到loss会在1B token左右迅速下降，之后随着训练token数量的增加，模型的能力会恢复的更多。一个很重要的事情是，我们的语料需要尽可能的包含更多的种类。

**Comment by Vincent131499** [comment]
ok, thanks

**Comment by pkumc** [comment]
> Thank you very much for your interest in our project. Regarding the specific fine-tuning of relu-llama, the related code and work will still require some time to be ready. If you are very interested, a basic way to reproduce our fine-tuning process is to simply change the activation function from silu to relu and then start training the model on the corpus. You will observe that the loss rapidly decreases around 1 billion tokens, and as the number of training tokens increases, the model's capabilities will be restored more and more. An important thing to note is that our corpus needs to include as many. :)
> 
> 非常感谢你对我们的关注，关于relu-llama的具体finetune，相关的代码和工作仍需要等待一段时间。如果你非常感兴趣，一个基本能复现我们finetune的方式就是直接将激活函数从silu改为relu后，开始模型在语料上的训练。你会看到loss会在1B token左右迅速下降，之后随着训练token数量的增加，模型的能力会恢复的更多。一个很重要的事情是，我们的语料需要更多元，包括更多不同方面的语料（code, knowledge等等）从而尽可能的多方面恢复模型的能力。

搭车同求！顺便问下：
